### PR TITLE
COMMERCE-2508 Fix Speedwell JS init

### PR DESCRIPTION
**Pull request by ethanbustad**
https://issues.liferay.com/browse/COMMERCE-2508

### CODE REVIEW

**Comment by ethanbustad** [comment]
Waiting on tests until https://github.com/liferay/com-liferay-commerce/pull/1749 is merged.

**Comment by marco-leo** [comment]
@gianmarcobrunialti

File: commerce-theme-speedwell/commerce-theme-speedwell/src/js/main.js [hunk 1]
```diff
@@ -1,12 +1,9 @@
 AUI().ready(
 	function() {
-		console.log('%c[ AUI ready ]', 'background-color: #000; color: #00FFFF');
-
-		if (!!Speedwell && !!Speedwell.features) {
+		if (!!Speedwell && !!Speedwell.features && !!Speedwell.features.init) {
 			Speedwell.features.init.initializeFeatures();
 
 			Speedwell.features.sliders = [];
-			Speedwell.features.sliderCallbacks = [];
```
Comment (gianmarcobrunialti) [inline commerce-theme-speedwell/commerce-theme-speedwell/src/js/main.js:9]: Why was this removed? It is necessary for every main.js reload to clean the queue of callbacks, if there are any. Up here it won't be useful, though. Let's move it right after the 'if'.

**Review by gianmarcobrunialti — CHANGES_REQUESTED** [review]

File: commerce-theme-speedwell/commerce-theme-speedwell/src/js/main.js (hunk 1 above)
Comment (ethanbustad) [reply commerce-theme-speedwell/commerce-theme-speedwell/src/js/main.js:9]: Ok. Yeah when it runs here it clears the queue right before checking whether there's anything in it. Makes sense to me, if we need to clean it, to clean it after it's used. Don't think I can make the change tonight - feel free to send a new pull with the correct changes tomorrow.

**Comment by marco-leo** [comment]
Fix included in  https://github.com/liferay/com-liferay-commerce/pull/1751
